config.go

```diff
@@ -171,6 +172,10 @@ type IterativeVerifierConfig struct {
 	//	1. Snappy (https://google.github.io/snappy/) as "SNAPPY"
 	//
 	// Optional: defaults to empty map/no compression
+	//
+	// Note that the IterativeVerifier is in the process of being deprecated.
+	// If this is specified, ColumnCompressionConfig should also be filled out in
+	// the main Config.
```

Comment:
I would like to get rid of the compression_verifier and iterative_verifier all together, but it would have to be another day. At that point I'll fix this comment.